### PR TITLE
Fix documentation related to number of sockets

### PR DESCRIPTION
**Pull request by bnellnm**
Some of the documentation wrt. number of sockets was out of date.  These changes should reflect the current state of things.

### CODE REVIEW

File: src/deepsparse/engine.py [hunk 1]
```diff
@@ -537,6 +533,8 @@ def benchmark_model(
     Gives defaults of batch_size == 1 and num_cores == None
     (will use all physical cores available on a single socket).
 
+    Note 1: Benchmarking is currenlty only supoorted on a single socket.
```
Comment (mgoin) [inline src/deepsparse/engine.py:536]:
```suggestion
    Note 1: Benchmarking is currently only supported on a single socket.
```

File: src/deepsparse/engine.py [hunk 2]
```diff
@@ -592,6 +590,8 @@ def analyze_model(
     Gives defaults of batch_size == 1 and num_cores == None
     (will use all physical cores available on a single socket).
 
+    Note 1: Analysis is currenlty only supoorted on a single socket.
```
Comment (mgoin) [inline src/deepsparse/engine.py:593]:
```suggestion
    Note 1: Analysis is currently only supported on a single socket.
```

**Review by mgoin — COMMENTED** [review]
LGTM just nits

File: src/deepsparse/engine.py (hunk 1 above)
Comment (bnellnm) [reply src/deepsparse/engine.py:536]: I really need new glasses :)

**Review by mgoin — APPROVED** [review]

**Review by bfineran — APPROVED** [review]

**Pull request merged**